### PR TITLE
Usability fixes following 4th test

### PR DESCRIPTION
**Pull request by jfhector**
I've left some comments (indicating why I made certain changes) in the code directly. (I'm not sure how else to provide comments in line for reviewers).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,143 +1,170 @@
...
+Your website/app must present information in ways people can recognise and use, no matter how they consume content (for example if they have low vision, or use assistive technologies like a screen reader).
 
 #### Provide text alternatives for images
 
-* [Text alternatives for images](guidelines/1.1.1.md): Provide a text description for images. Make sure the description serves the same purpose as the image.
+* [Text alternatives for images](guidelines/1.1.1.md): Provide a text description for images. Make sure the description conveys the same message or functionality.
+
+<!-- Note to reviewer: I'm experimenting with adding this bullet point, in spite of the fact that we want to keep the document as short as possible. Because this is one of the most frequent accessibility mistakes on the web, and a pain to screen reader users as they use the image's file name every time this happens. Also, this requirement is part of the success criterion, but is not included in the bullet point above. -->
+* [Decorative images](guidelines/1.1.1.md): If an image doesn't convey any information, make sure that this is indicated in code.
```
Comment (KaneCheshire) [inline README.md:20]: This bullet point breaks our new convention of making sure that the page that is linked to has the same heading and summary as the section it came from on the overview :(

File: README.md [hunk 2]
```diff
@@ -1,143 +1,170 @@
...
+* [Text or audio description (videos and presentations)](): For videos and presentations, provide an equivalent text description, or an audio description of what happens on screen.
 
-* [Audio description of pre-recorded videos or presentations](): If a video or presentation is prerecorded (rather than live), provide an audio description of the recording, even if you've already provided a text description.
+* [Audio description (videos presentations if pre-recorded)](): For videos and presentations that are pre-recorded, provide an audio description of what happens on screen, even if you've already provided a text description.
 
 #### Create content that can be presented in different ways
 
-* [Structure and relationships:](guidelines/1.3.1.md) When designs convey a structure of information visually (like sections within a page, or a data table), that structure must also be identified in code so that screen readers can understand it.
+* [Structure and relationships](guidelines/1.3.1.md): When designs convey a structure visually (like distinct sections within a page, or a label next to a checkbox) that structure must also be identified in code.
```
Comment (KaneCheshire) [inline README.md:34]: Need to make sure these changes are reflected in the details screens wherever they're trying to display this information verbatim

File: README.md (hunk 1 above)
Comment (jfhector) [reply README.md:20]: I was imagining that we would have 1 details page for 'Text alternatives for images', and another one for 'Decorative images'.

These two requirements are both summarised in 1.1.1 in the W3C guidelines, but they are different enough that we could make them 2 bullet points, with 2 detail pages.

This has downsides for sure (i.e. increased page length, and 2 detail pages linking to 1.1.1.). But this 'decorative images' requirement is largely unknown and causes lots of problems on the Web.

The alternative to having two bullet points would be to roll the point about decorative images into the bullet point above. We could. It'd just be quite long.

What do you think?

File: README.md (hunk 2 above)
Comment (jfhector) [reply README.md:34]: For sure, we will need to update the top of the details pages once we have the overview page in a more stable state.

**Pull request merged**